Simulator/Vertices/Neuro/AllIFNeurons_d.cpp

```diff
@@ -224,6 +224,6 @@ void AllIFNeurons::clearNeuronSpikeCounts(void *allVerticesDevice)
 ///  @param  edgeIndexMapDevice  GPU address of the EdgeIndexMap on device memory.
 void AllIFNeurons::advanceVertices(AllEdges &synapses, void *allVerticesDevice,
                                    void *allEdgesDevice, float *randNoise,
-                                   EdgeIndexMapDevice *edgeIndexMapDevice)
+                                   EdgeIndexMapDevice &edgeIndexMapDevice)
```

Comment:
Isn't this a GPU address? If so, it can't be a reference, because it actually isn't a pointer (which is an issue still hanging around that I hope to have someone work on).

Reply:
If this is true, make sure that no other GPU addresses have been converted to references by mistake.

Reply:
Changed it back to a pointer.